src/main/resources/db/changelog/db.changelog-RDM-9052.xml

```diff
@@ -0,0 +1,14 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+<!-- from version 3.7.0 onwards liquibase doesn't create the sequence unless the autoincrement is defined-->    
+    <changeSet id="RDM-9052-1" author="henry">
+	    <preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">select count (a.relname) from pg_class a where a.relname ='user_profile_audit' and Not Exists (SELECT c.relname FROM pg_class c WHERE c.relkind = 'S' and c.relname = 'user_profile_audit_id_seq')</sqlCheck>
+	    </preConditions>
+		<addAutoIncrement columnDataType="int" columnName="id" tableName="user_profile_audit"/>  
+	</changeSet>
+</databaseChangeLog>
+
+
```

Comment:
We might need to remove this RDM-9052 change? @henrykcgi

Reply:
not really it is the fix.